config/routes.rb

```diff
@@ -25,6 +25,8 @@
   # Articles Atom Feed
   get "feed", to: "articles#index", defaults: { format: "atom" }, as: :feed
 
+  # Articles - Collection Items
+  get "articles/:id/collection_posts", to: "collection_posts#index"
```

Comment:
I don't love `collection_posts` in the URL. But I'm also confused why this route is. 

Is this different than the route for an `Article`?